Emby.Drawing/Common/ImageHeader.cs

```diff
@@ -50,12 +50,18 @@ public static class ImageHeader
         /// <exception cref="ArgumentException">The image was of an unrecognised format.</exception>
         public static ImageSize GetDimensions(string path, ILogger logger, IFileSystem fileSystem)
         {
-            var extension = Path.GetExtension(path);
+            if (string.IsNullOrEmpty(path))
+            {
+                throw new ArgumentNullException("GetDimensions called without path");
+            }
 
-            if (string.IsNullOrEmpty(extension))
+            string extension = Path.GetExtension(path).ToLower();
+
+            if (extension.Length == 0)
```

Comment:
Unneeded check

Reply:
Filenames without dot (extensionless files) will then fall back to the next check and read "ImageHeader doesn't support " which I think is hard to understand

Reply:
removed it :)